# diff --git a/isotools-jessie/d-i/build/Makefile b/isotools-jessie/d-i/build/Makefile
# index 54a2fde..3c2da84 100644
# --- a/isotools-jessie/d-i/build/Makefile
# +++ b/isotools-jessie/d-i/build/Makefile
# @@ -496,7 +496,7 @@ ifndef ONLY_KLIBC
#  		-L $(TREE)/usr/lib/cdebconf/frontend \
#  		$(addprefix -l,$(notdir $(wildcard $(TREE)/usr/lib/cdebconf/frontend/*.so))) \
#  		`find $(TEMP) -type f -a \( -perm +0111 -o -name '*.so' -o -name '*.so.*' \) | \
# -		 grep -v udeblibs | grep -v 'usr/lib/xorg/modules/.*\.so'`
# +		 grep -v udeblibs | grep -v 'usr/lib/xorg/modules/.*\.so'`| grep -v 'ld\.so\.8\.gz'
 
#  ifeq ($(DEB_HOST_ARCH),armhf)
#  	# armhf has two linkers for now, while the world gets sorted
diff --git a/isotools-jessie/d-i/build/config/amd64/hd-media.cfg b/isotools-jessie/d-i/build/config/amd64/hd-media.cfg
index 9e1e49c..e3ace1d 100644
--- a/isotools-jessie/d-i/build/config/amd64/hd-media.cfg
+++ b/isotools-jessie/d-i/build/config/amd64/hd-media.cfg
@@ -4,7 +4,7 @@ FLAVOUR_SUPPORTED = "" gtk
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
 # and small enough to fit on a low-end memory stick, such as those
 # advertised as being 1 gigabyte in size.
-FLOPPY_SIZE = 976560
+FLOPPY_SIZE = 750000
 
 DISK_LABEL = "bootable drive"
 MEDIA_TYPE = bootable drive
# diff --git a/isotools-jessie/d-i/build/config/armel.cfg b/isotools-jessie/d-i/build/config/armel.cfg
# index e8fd439..ae39c68 100644
# --- a/isotools-jessie/d-i/build/config/armel.cfg
# +++ b/isotools-jessie/d-i/build/config/armel.cfg
# @@ -1,4 +1,4 @@
# -SUBARCH_SUPPORTED = iop32x kirkwood orion5x versatile ads
# +SUBARCH_SUPPORTED = versatile
 
#  KERNELMAJOR = 2.6
#  KERNELVERSION = 3.2.0-4
diff --git a/isotools-jessie/d-i/build/config/common b/isotools-jessie/d-i/build/config/common
index 9145ead..cac37e4 100644
--- a/isotools-jessie/d-i/build/config/common
+++ b/isotools-jessie/d-i/build/config/common
@@ -91,7 +91,8 @@ LSB_DISTRIB_RELEASE="$(DEBIAN_VERSION) - installer build $(BUILD_DATE)"
 
 # This keyring is only used to verify udeb downloads using apt. Since the
 # build process does not run as root, apt cannot read /etc/apt/trusted.gpg.
-KEYRING=/usr/share/keyrings/debian-archive-keyring.gpg
+#KEYRING=/usr/share/keyrings/debian-archive-keyring.gpg
+KEYRING=/etc/apt/trusted.gpg
 
 # All images that include cdebconf should include symbols needed by these
 # plugins.
diff --git a/isotools-jessie/d-i/build/config/i386/hd-media.cfg b/isotools-jessie/d-i/build/config/i386/hd-media.cfg
index 9e1e49c..e3ace1d 100644
--- a/isotools-jessie/d-i/build/config/i386/hd-media.cfg
+++ b/isotools-jessie/d-i/build/config/i386/hd-media.cfg
@@ -4,7 +4,7 @@ FLAVOUR_SUPPORTED = "" gtk
 # netinst iso in (being able to fit the full CD on it is just a bonus..),
 # and small enough to fit on a low-end memory stick, such as those
 # advertised as being 1 gigabyte in size.
-FLOPPY_SIZE = 976560
+FLOPPY_SIZE = 750000
 
 DISK_LABEL = "bootable drive"
 MEDIA_TYPE = bootable drive
diff --git a/isotools-jessie/debian-cd/CONF.sh b/isotools-jessie/debian-cd/CONF.sh
index 11c613d..ffea236 100644
--- a/isotools-jessie/debian-cd/CONF.sh
+++ b/isotools-jessie/debian-cd/CONF.sh
@@ -59,7 +59,7 @@ export BASEDIR=`pwd`
 # export CDNAME=debian
 
 # Building wheezy cd set ...
-export CODENAME=wheezy
+export CODENAME=jessie
 
 # By default use Debian installer packages from $CODENAME
 if [ -z "$DI_CODENAME" ]; then
--- a/isotools-jessie/debian-cd/CONF.sh
+++ b/isotools-jessie/debian-cd/CONF.sh
@@ -76,7 +76,7 @@ fi
 #export DI_WWW_HOME=default
 
 # Version number, "2.2 r0", "2.2 r1" etc.
-export DEBVERSION="7.0"
+export DEBVERSION="8.0"
 
 # Official or non-official set.
 # NOTE: THE "OFFICIAL" DESIGNATION IS ONLY ALLOWED FOR IMAGES AVAILABLE
@@ -115,7 +115,7 @@ export MIRROR=/srv/mirror/debian
 export TDIR=/srv/mirror/tmp
 
 # Path where the images will be written
-export OUT=/srv/mirror/debian-cd-test
+export OUT=/tmp/iso-images
 
 # Where we keep the temporary apt stuff.
 # This cannot reside on an NFS mount.
diff --git a/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86 b/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
index c6d397c..b523270 100644
--- a/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
+++ b/isotools-jessie/debian-cd/tools/boot/jessie/boot-x86
@@ -429,6 +429,11 @@ if [ -n "$KERNEL_PARAMS" ]; then
     done
 fi
 
+rm -f boot$N/isolinux/{*txt,*cat,ad*,exit*,prompt*,rq*,spk*}
+cp -f ../../../cd-root/isolinux/* boot$N/isolinux/
+rm -fr CD$N/README.*
+cp -r ../../../cd-root/* CD$N/
+
 set_default_desktop
 rm -rf cdrom
 
diff --git a/isotools-jessie/d-i/build/boot/x86/gtk.cfg b/isotools-jessie/d-i/build/boot/x86/gtk.cfg
index a0592c4..330edd4 100644
--- a/isotools-jessie/d-i/build/boot/x86/gtk.cfg
+++ b/isotools-jessie/d-i/build/boot/x86/gtk.cfg
@@ -1,4 +1,5 @@
 label installgui
 	menu label ^Graphical install
+	menu default
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE_GTK} initrd=${INITRD_GTK} --- quiet ${CONSOLE}
diff --git a/isotools-jessie/d-i/build/boot/x86/txt.cfg b/isotools-jessie/d-i/build/boot/x86/txt.cfg
index 04adb7d..22cecea 100644
--- a/isotools-jessie/d-i/build/boot/x86/txt.cfg
+++ b/isotools-jessie/d-i/build/boot/x86/txt.cfg
@@ -1,6 +1,5 @@
 default install
 label install
 	menu label ^Install
-	menu default
 	kernel ${KERNEL}
 	append desktop=%desktop% ${VIDEO_MODE} initrd=${INITRD} --- quiet ${CONSOLE}
